packages/example-getting-started/config/datasources.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ds",
+  "name": "todo",
```

Comment:
Please note that a single datasource can back multiple models, therefore the datasource name is very rarely the same as the model name.

In LoopBack 3.x, we were using `db` as the name of the "main" datasource. Could you please use "db" in this example project too?

Reply:
Yes, we should keep it consistent to `db`